src/App/Pages/Accounts/LoginPage.xaml.cs

```diff
@@ -25,6 +26,7 @@ public LoginPage(string email = null, AppOptions appOptions = null)
             _vm.Page = this;
             _vm.StartTwoFactorAction = () => Device.BeginInvokeOnMainThread(async () => await StartTwoFactorAsync());
             _vm.LogInSuccessAction = () => Device.BeginInvokeOnMainThread(async () => await LogInSuccessAsync());
+            _vm.LogInSSOCommand = new Command(async () => await StartSsoLoginAsync());
```

Comment:
Change this to action "pattern" as the others so that it can be overriden from the iOS extensions.
Maybe we can add a tech-debt task to refactor the pattern into something better, but for now maybe we could stick to it so it's easier to maintain.